subprojects/docs/docs.gradle

```diff
@@ -75,7 +75,7 @@ dependencies {
         exclude group: 'org.apache.httpcomponents'
     }
     testCompile libraries.commons_httpclient
-    testCompile 'org.apache.httpcomponents:httpmime:4.4.1'
+    testCompile 'org.apache.httpcomponents:httpmime:4.5.5'
```

Comment:
`"org.apache.httpcomponents:httpmime:${versions.httpclient}"` to keep them in sync or do they just happen to have the same version now?

Reply:
This is just for running tests in the `docs` module. I just upgraded the version to keep them in sync.

Reply:
Used the extra property.